build/pkgs/_prereq/distros/conda.txt

```diff
@@ -1,7 +1,8 @@
-compilers
-make
-m4
-perl
 python
-tar
-bc
+# Not needed on conda
```

Comment:
it would only make sense to me if the packages listed there are always included with conda, but that's not the case AFAIK.

Reply:
It's a long time since I did this change. I don't quite remember why I removed them, there was some problem. Most of these (all?) are only prerequisites of sage-the-distro, and not sagelib-the-python-package. With conda, we don't use make or untar things. (Not so sure about perl, bc and compilers, but they don't seem to be needed/are included anyway)

Reply:
I don't quite understand what you mean by this. Could you please rephrase it?

Looking at the ci (eg https://github.com/sagemath/sage/actions/runs/7003785707/job/19050185446?pr=35986), one sees that the build + test is successful without bc and tar (c and fortran compilers, make, m4, perl are still pulled-in by other dependencies). But as I said above, I don't remember why I've removed these. Should I try to readd all of them, or are there indeed packages that are clearly not needed as actual prerequisites for sagelib?

Reply:
Dima is right here, these packages cannot be removed. `build/pkgs/_prereq/distros/conda.txt` are the minimal requirements for building the Sage distribution from source; not just the Sage library. This is not tested in ci-conda, but it is tested using the CI Linux workflow. See https://github.com/sagemath/sage/actions/runs/7076264535/job/19259311074 -- note there is a build failure (needs #23712)

Reply:
I've removed these changes again. It would be good to clarify the purpose of `_prereq`, since it says `This dummy package represents the minimal requirements (system packages)
for installing SageMath from source.` which is not really what it does according to the discussion above, at least if you understand SageMath = sagelib. Most of these dependencies should then be added directly to sagelib, and then we can exclude `_prereq` from the env files under `src`. Let's leave this for another PR.